pkg/grizzly/parsing.go

```diff
@@ -133,7 +133,11 @@ func ParseJsonnet(registry Registry, jsonnetFile string, opts Opts) (Resources,
 		if err != nil {
 			return nil, err
 		}
-		resources = append(resources, parsedResources...)
+		for _, parsedResource := range parsedResources {
+			if parsedResource.MatchesTarget(opts.Targets) {
+				resources = append(resources, parsedResource)
+			}
+		}
```

Comment:
somehow we lost our `-t` target handling functionality when parsing with Jsonnet.